flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/catalog/CatalogTableITCase.scala

```diff
@@ -919,6 +919,20 @@ class CatalogTableITCase(isStreamingMode: Boolean) extends AbstractTestBase {
       .getTable(new ObjectPath(tableEnv.getCurrentDatabase, "t2"))
       .getProperties
     assertEquals(expectedProperties, properties)
+    val currentCatalog = tableEnv.getCurrentCatalog
+    val currentDB = tableEnv.getCurrentDatabase
+    tableEnv.sqlUpdate("alter table t2 add constraint ct1 primary key(a) not enforced")
```

Comment:
This should fail? Because `a` is nullable.

Btw, please use `tableEnv.executeSql(...)`.

Reply:
Is it to test for the exception? I didn't see the catched exception here.

Reply:
If we change a to not null, it would throws, there is a test in SqlToOperationConverterTest